exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/DirectScanPrel.java

```diff
@@ -31,26 +36,23 @@
 import org.apache.drill.exec.physical.base.GroupScan;
 import org.apache.drill.exec.physical.base.PhysicalOperator;
 import org.apache.drill.exec.physical.base.ScanStats;
-import org.apache.drill.exec.planner.cost.DrillCostBase;
 import org.apache.drill.exec.planner.fragment.DistributionAffinity;
 import org.apache.drill.exec.planner.physical.visitor.PrelVisitor;
 import org.apache.drill.exec.record.BatchSchema;
 
-import java.io.IOException;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import org.apache.drill.exec.planner.cost.DrillCostBase.DrillCostFactory;
 
+//public class DirectScanPrel extends AbstractRelNode implements DrillScanPrel
 public class DirectScanPrel extends AbstractRelNode implements Prel, HasDistributionAffinity {
-  static final org.slf4j.Logger logger = org.slf4j.LoggerFactory
-          .getLogger(DirectScanPrel.class);
+  static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DirectScanPrel.class);
 
   private final GroupScan groupScan;
   private final RelDataType rowType;
 
   DirectScanPrel(RelOptCluster cluster, RelTraitSet traits,
                  GroupScan groupScan, RelDataType rowType) {
     super(cluster, traits);
+    //this.groupScan = getCopy(groupScan);
```

Comment:
If you think previously the copy of groupScan was unnecessary then we should just remove this line...but pls check the history of this since there may be legitimate reason not to share the groupScan instance.

Reply:
This is just a comment now. Removed it.